### ISSUE TITLE
Error given when doing "python buy_order_bot.py" in terminal

### ISSUE THREAD

**Issue by Cres543** [YESW0RLD/BuyOrderBot#2]
![image](https://github.com/YESW0RLD/BuyOrderBot/assets/133691171/c64dd211-14e4-468d-a0b2-038697375585)
I've tried to reinstall it many times and the error still happens. Any ways to fix this?? Thanks!

**Comment by YESW0RLD** [comment]
Have you installed the newest version of steampy?

**Comment by Cres543** [comment]
Yes i did but its not in the folder of the bot, its in a different file, do I have to move it anywhere?

**Comment by YESW0RLD** [comment]
It would be good, but you also can import files from a different folders.

**Comment by Cres543** [comment]
Sorry I don't get what you mean, what files do I need to import from steampy folder to the buy order bot folder?

Sorry

**Comment by YESW0RLD** [comment]
Library files, modules, they are files, simply saying

**Comment by Cres543** [comment]
I see what you are on about now! Its telling me to do the login method for steampy, can you tell me how I can get the path to my steamguard file? I have steam desktop authenticator. 

Thanks

**Comment by YESW0RLD** [comment]
Because you are using SDA, you can save shared_secret and identity_secret in file with name steam_guard.json. There is steps you need to do to setup bot in README.[https://github.com/YESW0RLD/BuyOrderBot/blob/master/README.md](README)

**Comment by Cres543** [comment]
Is it possible if i can add your discord? It will be easier to communicate there

**Comment by Cres543** [comment]
killer.flakez add this please on discord :)

**Comment by YESW0RLD** [comment]
Yes, yiu can find it in my profile.

**Comment by Cres543** [comment]
The link on your profile doesn't work, can you tell me your name so I can just add you?

**Comment by YESW0RLD** [comment]
Everything works well.

**Comment by Cres543** [comment]
The link to your discord account ("https://discordapp.com/users/357156620028870668")  does not work, friend.

**Comment by woctezuma** [comment]
It works for me.

**Comment by yesworId** [comment]
Yeah mate, just login in your discord through the browser or open it on your pc. I don't know what the problem you are dealing with and do don't know what do you want.

**Comment by Cres543** [comment]
I can't add your discord, but look. 
![image](https://github.com/yesworId/BuyOrderBot/assets/133691171/3b3017a3-a10a-4e96-b19a-eb0ab4ea3a61)
I keep on getting this error when I do "python buy_order_bot.py", I have edited every file except "cookies.json", with the correct information, like my shared secret and identity secret which I got from "steam desktop authenticator", I don't know what to do next.

**Comment by Cres543** [comment]
I have steampy installed from when you told me to download it around 13 hours ago

**Comment by yesworId** [comment]
-You don't need to create cookies file an edit it. Program will automatically do this. 
-You don't need to change config or steam_guard files, cuz you are getting TypeError in SteamClient init function.
-If you are getting the same error, as shown on the screen: I suppose this is because you are using wrong steampy version. You need to install the newest one. You can find version in requirements.
